### ISSUE TITLE
[Extractor] Make default value in target

### ISSUE THREAD

**Issue by xfifix** [php-translation/symfony-bundle#235]
Since i switched of symfony/translation bundle to php-translation because i needed to extract key of php. Now i don't have default value in target any more.
I would like to know if it's possible to have a default value in target tags instead of empty ? :)

**Comment by xfifix** [comment]
i created a PR #238

**Comment by Nyholm** [comment]
Thank you for the issue and PR. 
Could you elaborate on the use case for this?

**Comment by xfifix** [comment]
because we don't have the translation yet so it's good to have a default string to test UI.

**Comment by Nyholm** [comment]
That is what the ‘@desc’ annotation is for. Right?

**Comment by xfifix** [comment]
i tried your PR and the target stay empty. I would like the target to be auto generated with the data source.

**Comment by Nyholm** [comment]
This will extract a key "foobar" with the translation "English translation". 

```php
@desc("English translation")
$this->translator->trans('foobar')
```

--------
Current behavior is that this extracts a key "foobar" but with no translation.

```php
$this->translator->trans('foobar')
```

--------
Do I understand you correctly when you want the following to extract a key "foobar" with translation "foobar"?

```php
$this->translator->trans('foobar')
```

**Comment by xfifix** [comment]
yes if it's new translation like the default translation symfony. The default translator's symfony generate the translation : _foobar if  the id translation is new. 
That is created automatically the translation data for my UI during his developpement.

**Comment by pascalwacker** [comment]
A quick and dirty solution is to run the following JS in the Translation Web UI:
```
document.querySelectorAll('textarea.form-control[data-key]').forEach(function(a) {
  if (!a.value) {
    a.value = a.dataset.key;
    a.onchange();
  }
});
```

What would you think about adding a button like "Replace Empty Values with default values"? I'm not sure if it makes sense, as you might have one or two entries that are an empty string.
